socket.js

```diff
@@ -1,6 +1,16 @@
 import { Server } from "socket.io";
 import { updateSocketId } from "./Controller/user.js";
+import { newMessage } from "./Controller/message.js";
 
+import { v2 as cloudinary } from "cloudinary";
+
+cloudinary.config({
+  cloud_name: "dubwggg4a",
+  api_key: "431517811498724",
+  api_secret: "eoejYyKRwg1jg6EBHzh2evyjkEE",
```

Comment:
@athar-shayan I've previously mentioned in your previous pull requests to store sensitive information such as `cloud_name`, `api_key`, and `api_secret` in your .env files. Please make sure to move these credentials to the .env files.

Reply:
Done.

Reply:
@athar-shayan Currently, you are accessing the values directly from the .env file. Sir Arslan suggested creating a constant.js file and importing these constants into that file abd then use it.